### PR TITLE
Tp #41 update seed file

### PR DESCRIPTION
**Pull request by Noemi890**
Updated seed file with more students and more posts

### CODE REVIEW

File: prisma/seed.js [hunk 1]
```diff
@@ -30,41 +30,216 @@ async function seed() {
...
+
+  const ninthUser = await prisma.user.create({
+    data: {
+      email: 'ninth@blah',
+      password,
+      cohortId: cohort1.id
+    }
+  })
+
+  const tenthUser = await prisma.user.create({
```
Comment (vherus) [inline prisma/seed.js:89]: That's a lot of repetition!

There are two ways I think you could really improve this.

1. Use Prisma's `createMany` function to create 10 users. You can run a `findMany` afterwards to get all of their ID's - though this still has the problem of having a lot of lines of code where you're defining different email addresses

2. (Preferable) Use a loop to `create` 10 users, their profiles and their posts in one query, using the index to create unique email addresses. You can save each created user (include their profile and posts in the returned data from prisma) into an array during the loop, then run another loop to create some likes.

What do you think?

Comment (Noemi890) [reply prisma/seed.js:89]: Yeah I think I will go for the second option, I will improve this and get back to you when it is done!!!

File: prisma/seed.js [hunk 2]
```diff
@@ -6,27 +6,62 @@ const prisma = new PrismaClient()
...
+    'https://images.unsplash.com/photo-1574144611937-0df059b5ef3e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=928&q=80',
+    'https://images.unsplash.com/photo-1571566882372-1598d88abd90?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=774&q=80',
+    'https://images.unsplash.com/photo-1506891536236-3e07892564b7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=776&q=80',
+    'https://images.unsplash.com/photo-1511275539165-cc46b1ee89bf?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80',
+    'https://images.unsplash.com/photo-1572252821143-035a024857ac?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=772&q=80'
+  ]
+
+  for (let i = 0; i <= 9; i++) {
+    if (i <= 3) {
+      const cohort = await prisma.cohort.create({
```
Comment (vherus) [inline prisma/seed.js:26]: Nice!

**Review by vherus — APPROVED** [review]
Much better! Great work

**Review by miresque — APPROVED** [review]

**Pull request merged**